templates/download/server/arm.html

```diff
@@ -101,9 +101,9 @@ <h2>
         <li class="p-list__item is-ticked">
           Application container technology based on Docker and Kubernetes, including FAN-based networking.
         </li>
-        <li class="p-list__item is-ticked">SQL and NoSQL databases including CouchDB, MySQL, PostgreSQL, Redis and SQLite.</li>
+        <li class="p-list__item is-ticked">SQL and NoSQL databases including MySQL, PostgreSQL, Valkey and SQLite.</li>
         <li class="p-list__item is-ticked">
-          A wide collection of web technologies, from HTTP servers to frameworks including Apache, Django, memcached, Nginx, WordPress and Varnish.
+          A wide collection of web technologies, from HTTP servers to frameworks including Apache, Nginx, Django, memcached and Varnish.
```

Comment:
The discussion in the copydoc ended by please removing Varnish from that list.
Please drop it here